test/test_jit.py

```diff
@@ -2213,6 +2213,41 @@ def forward(self, v):
         with self.assertRaisesRegex(RuntimeError, "cannot be used as tuple"):
             M()
 
+    def test_rnn_trace_override(self):
+        from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence
+        T, B, C = 3, 5, 7
+
+        class PadPackedWrapper(torch.nn.Module):
+            def __init__(self):
+                super(PadPackedWrapper, self).__init__()
+
+            def forward(self, x, seq_lens):
+                x = pack_padded_sequence(x, seq_lens)
+                x, _ = pad_packed_sequence(x)
+                return x
+
+        x = torch.ones(T, B, C, requires_grad=True)
+        seq_lens = torch.ones(B, dtype=torch.int32)
```

Comment:
Can we test on some other tensor? This one is extremely simple, and it would also be good to assert that the output is equal to input (since those functions are meant to be inverses of each other).